content/getting-started/setup/upgrades/suggested-upgrade-path/index.md

```diff
@@ -0,0 +1,32 @@
+---
+uid: setup-upgrades-suggested-upgrade-path
+locale: en
+title: DNN Platform Suggested Upgrade Path
+dnnversion: 09.04.03
+---
+
+# DNN Platform Suggested Upgrade Path
+Following is the recommend upgrade path for **DNN Platform** based on experiences of many DNN Community members.
+
+> [!WARNING]
+> While the below has been tested and used successfully, there are no guarantees. Each upgrade scenario can result in unique complexities and challenges. Therefore, each upgrade should be treated with full context in mind. For common best-practices, please use [these basic upgrade steps](xref:setup-upgrades) to provide guidance.
+
+The below is, of course, subject to change. If your current version of DNN Platform is between any of the versions listed below, first upgrade to the closest listed version.
```

Comment:
For instance, if you are starting with version 09.02.00, you should upgrade first to version 09.03.02.

Reply:
Okay, good idea.  I'll create another PR for this.